README.md

```diff
@@ -1,21 +1,20 @@
 # OpenSwap
 
-*Disclaimer: I am completing this project with students at Seven Academy to illustrate to them fundermental backend and express concepts*
+_Disclaimer: I am completing this project with students at Seven Academy to illustrate to them fundermental backend and express concepts_
```

Comment:
This was not added by you, why does it appear as if you added it?

I should just see additions or modifications